### ISSUE TITLE
False positive in operator_usage_whitespace

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#1648]
```swift
// shouldn't trigger
let red  = CGFloat((rgbaValue >> 24) & 0xff) / 255.0
```

**Comment by marcelofabri** [comment]
Actually, this is triggering because of the extra space, not `>>` or `<<` 🤦‍♂️
